### DISCUSSION TITLE
How to hover and scroll (mouse wheel action)?

### DISCUSSION THREAD

**Discussion by N00Boj** [cymian/mouseless#70] [Q&A]
Hi, love the concept and implementation of your app!

I'm trying it out for the first time today, but can't figure out how to scroll, say, a website or document without switching focus to it's window and/or clicking on it. With a mouse you'd hover over the window and use the mouse wheel. Is there a way to do this and I just haven't found it yet?

Cheers and thanks for your great work
oj

**Comment by cymian** [comment]
Hi @N00Boj!  Thank you so much for the kind words.  I really appreciate it!

Yep, will be adding this very soon!  Probably the release after this week's.  You can track it on issue #21 (so I will close this discussion in favor of that one).

In the meantime, maybe arrow keys and  spacebar / shift+spacebar can help you out in some situations.

Thanks again, and let me know if you have any more questions!
